README.md

```diff
@@ -82,19 +82,32 @@ It can be either a relative or absolute path.
 ## Testing
 
 The SDK has integration tests which run against a real Chatkit server.
-In order to run them, you must provide Chatkit instance credentials in
-your Gradle `local.properties` (or as `-D` arguments to the VM executing
-the tests)
 
-*Important:* The tests may delete any and all resources associated with
-the instance you provide. Create a new one. Do not share it with any other
-use.
+The tests can be run using the Gradle `test` task with the VM options mentioned below set.
+To run the tests directly from Android Studio you will need to install the
+[Spek](https://plugins.jetbrains.com/plugin/8564-spek/) plugin, and
+[Spek Framework](https://plugins.jetbrains.com/plugin/10915-spek-framework/).
+Both plugins are needed as older tests are written using Spek1, and newer
```

Comment:
The authors call it: Spek 1 (with space), precisely they write Spek `1.x` and Spek `2.x`. both on top of their [page](https://www.spekframework.org/) and [GH](https://github.com/spekframework/spek/#current-state-of-the-project). I think we should follow the same naming (with space so that Spek is a word), just skipping the monospaced formatting and `.x` for ver no as that's more typographical/special style.